ogame/constants.py

```diff
@@ -128,6 +164,24 @@ def is_research(research):
         else:
             return False
 
+    def research_name(res):
```

Comment:
when i first coded this i didnt understand hasmaps.
but today i know them.
when i have the time i should convert all const to hasmaps that are implementet in if chains